autoload/ale/fixers/rubocop.vim

```diff
@@ -0,0 +1,9 @@
+function! ale#fixers#rubocop#Fix(buffer) abort
+  let l:executable = ale_linters#ruby#rubocop#GetExecutable(a:buffer)
+
+  return {
+        \   'command': ale#Escape(l:executable)
+        \       . ' --auto-correct %t',
+        \   'read_temporary_file': 1,
+        \}
+endfunction
```

Comment:
Your indentation needs to be fixed here to match the other files. The test output mentions this somewhat.